### PR TITLE
updated the read me to include setting up admin

### PR DESCRIPTION
**Pull request by sewagodimo**
I updated the read me to include some guides on setting up the app for the surveys

### CODE REVIEW

**Comment by sewagodimo** [comment]
hey @codiebeulaine I updated the read me to include some information that I found useful for setting up

**Comment by nathanbegbie** [comment]
@sewagodimo when working with RST, using the `$` symbol is used to indicate commands to run or code. It's better to structure your instructions as normally formatted text.

File: README.rst [hunk 1]
```diff
@@ -6,15 +6,22 @@ This is an application scaffold for Molo_.
 Getting started
 ---------------
 
-To get started::
+To set up enviroment::
 
-    $ virtualenv ve
+    $ virtualenv vegit pu
```
Comment (nathanbegbie) [inline README.rst]: I don't think this should be changed.

File: README.rst [hunk 2]
```diff
@@ -6,15 +6,22 @@ This is an application scaffold for Molo_.
...
     $ pip install -e .
+    $ in a new terminal: redis-server
```
Comment (nathanbegbie) [inline README.rst]: Rather separate this into 2 lines:
```
In a seperate terminal::

    $ redis-server
```

File: README.rst [hunk 3]
```diff
@@ -6,15 +6,22 @@ This is an application scaffold for Molo_.
...
     $ ./manage.py runserver
 
 You can now connect access the demo site on http://localhost:8000
 
+To get started::
+	$ login to : http://localhost:8000/admin
+	$ click on settings, then choose language
+	$ add a new language and save it
+	$ click settings and choose site settings
+	$ tick all the ARTICLE TAG SETTINGS (Display tags on front ent and enable tag navigation)
```
Comment (nathanbegbie) [inline README.rst]: Rather format this as a list, not as code.

**Comment by sewagodimo** [comment]
@nathanbegbie I added the changes you requested. Thank you.

File: README.rst [hunk 4]
```diff
@@ -16,5 +19,11 @@ To get started::
 
 You can now connect access the demo site on http://localhost:8000
 
+To get started::
+	* login to : http://localhost:8000/admin
```
Comment (nathanbegbie) [inline README.rst]: `login to :` -> `log in to:`

File: README.rst [hunk 5]
```diff
@@ -16,5 +19,11 @@ To get started::
...
+	* click on settings, then choose language
+	* add a new language and save it
+	* click settings and choose site settings
+	* tick all the ARTICLE TAG SETTINGS (Display tags on front ent and enable tag navigation)
```
Comment (nathanbegbie) [inline README.rst]: `ent` -> `end`

File: README.rst [hunk 6]
```diff
@@ -5,8 +5,11 @@ This is an application scaffold for Molo_.
 
 Getting started
 ---------------
+In a seperate terminal::
```
Comment (codiebeulaine) [inline README.rst]: seperate should be separate

**Comment by codiebeulaine** [comment]
Thanks for adding this @sewagodimo 🍰 
Just a few comments but looks good

**Comment by codiebeulaine** [comment]
👍 🍰

**Comment by sewagodimo** [comment]
@nathanbegbie  and @codiebeulaine I made the changes. Thank you guys for the reviews

**Pull request merged**